src/views/Correction/BenCorrection.vue

```diff
@@ -117,6 +118,9 @@ export default class BenCorrection extends Mixins(CommonMixin, DateMixin, FeeMix
       // initialize Fee Summary data
       this.setFilingData([this.correctionResource.filingData])
 
+      // pre-select No Fee option
+      this.setStaffPayment({ option: StaffPaymentOptions.NO_FEE })
+
```

Comment:
Firm corrections do this explicitly, so I'm assuming that BEN corrections should do this, too.